docs/platform/connector-development/connector-builder-ui/authentication.md

```diff
@@ -17,8 +17,12 @@ Check the documentation of the API you want to integrate for the used authentica
 - [Basic HTTP](#basic-http)
 - [Bearer Token](#bearer-token)
 - [API Key](#api-key)
-- [OAuth](#oauth)
+- [JWT](#jwt)
+- [OAuth 2.0 (Declarative)](#oauth-20-declarative)
+- [OAuth (Legacy)](#oauth-legacy)
```

Comment:
```suggestion
- [OAuth2](#oauth-2)
```
The new UI has actually merged the declarative and legacy OAuths into one - now users can utilize Declarative OAuth by selecting the OAuth2 option and then enabling `DeclarativeOAuth Connector Specification` inside it.